src/server/models/user.ts

```diff
@@ -16,38 +16,44 @@ export interface User extends Document {
   refreshToken: string;
   validatePassword(hash: string): Promise<boolean>;
   isAllowed(requiredRoles: Role[]): boolean;
+  toPresentationLayer(): Thruway.User;
```

Comment:
We must make sure we get `Thruway` namespaced interfaces when invoking `toPresentationLayer` as these are used by the _client side_ as well, and must be the returned ones from every API route